src/components/ToolbarOverlay.tsx

```diff
@@ -331,6 +334,13 @@ export const ToolbarOverlay = ({ children, elements }: Props) => {
                     setIsShowingAutorouting(!is_showing_autorouting)
                   }}
                 />
+                <CheckboxMenuItem
+                  label="Show DRC Errors"
+                  checked={is_showing_DRC_errors}
+                  onClick={() => {
+                    setIsShowingDRCErrors(!is_showing_DRC_errors)
```

Comment:
```suggestion
                    setIsShowingDrcErrors(!is_showing_drc_errors)
```

we use google-casing https://github.com/tscircuit/handbook/blob/main/guides/code.md#4-casing